src/manifest_store.rs

```diff
@@ -596,4 +676,79 @@ mod tests {
         let result = ms.delete_manifest(2).await;
         assert!(matches!(result, Err(error::SlateDBError::InvalidDeletion)));
     }
+
+    fn new_memory_manifest_store() -> Arc<ManifestStore> {
+        let os = Arc::new(InMemory::new());
+        Arc::new(ManifestStore::new(&Path::from(ROOT), os.clone()))
+    }
+
+    fn now_rounded_to_nearest_sec() -> SystemTime {
+        let now_secs = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .unwrap()
+            .as_secs();
+        SystemTime::UNIX_EPOCH + Duration::from_secs(now_secs)
+    }
+
+    #[tokio::test]
+    async fn test_read_active_manifests_should_consider_checkpoints() {
```

Comment:
this test looks similar to `test_read_active_checkpoints`. What is the additional aspect validated in this test?

Reply:
Yes, the tests seem redundant. Let me try to consolidate them.